packages/components/src/templates/next/components/complex/DynamicStatistics/DynamicStatistics.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { withChromaticModes } from "@isomer/storybook-config"
+
+import { DynamicStatisticsUI } from "./DynamicStatisticsClient"
+
+const meta: Meta<typeof DynamicStatisticsUI> = {
+  title: "Next/Components/DynamicStatistics",
+  component: DynamicStatisticsUI,
+  parameters: {
+    layout: "fullscreen",
+    chromatic: withChromaticModes(["mobile", "tablet", "desktop"]),
+  },
+}
+
+export default meta
+type Story = StoryObj<typeof DynamicStatisticsUI>
+
+export const Default: Story = {
+  args: {
+    title: "1 Rejab 1446H",
+    statistics: [
+      { label: "Subuh", value: "5:43am" },
```

Comment:
nit: i don't think they add the "am" or "pm" at the back looking at muis website

Reply:
yeah tho i do agree it's clearer + otherwise the UI will be too too empty 🥲

Reply:
I added this in design because dates are generally more readable with am/pm, and 24-hour format (3rd alternative) is not super laymen friendly!

Reply:
okay sounds good